backend/src/api.py

```diff
@@ -268,8 +301,12 @@ def _refresh_nodes(self):
                         self.nodes[node.schema_id] = node, sub
 
 
+# pylint: disable=global-at-module-level
+global registry
```

Comment:
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/global-at-module-level.html
> Used when you use the "global" statement at the module level since it has no effect.

I don't think we need this then.